doc/contributing/file-structure.chapter.md

```diff
@@ -0,0 +1,24 @@
+# File Structure {#chap-file-structure}
+
+Nixpkgs is roughly organized in the following structure:
+
+- `doc`: Sources and infrastructure for this manual, see also [](#chap-contributing)
+- `lib`: Library functions and inlined sources for [](#part-lib)
+- `nixos`: NixOS, see the [Development section in the NixOS manual](https://nixos.org/nixos/manual/#ch-development)
+- `pkgs`: Entry-points, packages, builders
+  - `top-level`: Entrypoints, package set aggregations
+    - `impure.nix`, `default.nix`, `config.nix`: Root entry-point (`import <nixpkgs>`) function definition
+    - `stage.nix`, `all-packages.nix`, `named-hierarchy-files.nix`, `splice.nix`: Top-level attribute set (`import <nixpkgs> {…}`) aggregation definition
```

Comment:
```suggestion
    - `stage.nix`, `all-packages.nix`, `named-hierarchy-files.nix`, `splice.nix`: Top-level attribute set (`import <nixpkgs> {…}`) aggregation definition also known as `pkgs`
```

Reply:
This was part of https://github.com/NixOS/nixpkgs/pull/245243, though it was changed a bit: https://github.com/NixOS/nixpkgs/tree/b978241b83a7702dc2678d97dd03789073ca715c/pkgs#overview, I'll resolve this